src/style.css

```
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+form {
+  position: relative;
+  margin-top: 20px;
+}
+
+input {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  background-color: #fff;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background-color: #e5e5e5;
+  line-height: 22px;
+  font-weight: bold;
+  font-size: 16px;
+}
+
+input::placeholder {
+  color: #3d4e61;
+}
+
+#bigInput {
+  padding: 24px 0 24px 62px;
+  height: 70px;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+#smallInput {
+  padding: 12px 0 12px 33px;
+  height: 42px;
+  font-size: 14px;
+  font-weight: 300;
+}
+
+.bigIcon {
+  position: absolute;
+  top: 25px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.smallIcon {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+#smallInput:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+#smallInput:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  line-height: 19.12px;
```

I know figma mock has this decimal measure, but when you take a look at a `Code` section under `Typography` - you'll see it's set to 19px
avoid decimals for absolute measures
```suggestion
  line-height: 19px;
```